home/models.py

```diff
@@ -209,10 +209,17 @@ def get_context(self, request):
             self.featured_content.filter(content__live=True)
         ]
         context['sub_sections'] = self.get_children().live().type(Section)
+
         context['articles'] = self.get_children().live().type(Article)
-        context['surveys'] = self.get_children().live().type(Survey)
-        context['polls'] = self.get_children().live().type(Poll)
-        context['quizzes'] = self.get_children().live().type(Quiz)
+
+        survey_page_ids = self.get_children().live().type(Survey).values_list('id', flat=True)
```

Comment:
shouldn't we filter out polls, surveys, and quizzes which are multi-step as multi-step forms don't work in direct display?

Reply:
Good point. I think the better way to do this would be to add validation in the Admin page. To not allow direct display for multistep forms. Will make a ticket

Reply:
#329